src/App.tsx

```diff
@@ -10,7 +12,15 @@ interface State {
 
 class App extends React.Component<{}, State> {
   state: State = {
-    selectedUserId: 0,
+    selectedUserId: 1,
+  };
```

Comment:
Weird to see this on the start, just set null, and if it is null render smth like `No user selected'
![image](https://user-images.githubusercontent.com/37161315/147341552-c025afa9-f583-4bf6-b382-45572b1debb5.png)